src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -42,6 +42,14 @@ class StableDiffusionPipeline(DiffusionPipeline):
             Model that extracts features from generated images to be used as inputs for the `safety_checker`.
     """
 
+    vae: AutoencoderKL
+    text_encoder: CLIPTextModel
+    tokenizer: CLIPTokenizer
+    unet: UNet2DConditionModel
+    scheduler: Union[DDIMScheduler, PNDMScheduler, LMSDiscreteScheduler]
+    safety_checker: StableDiffusionSafetyChecker
+    feature_extractor: CLIPFeatureExtractor
```

Comment:
Just a helper for IDEs, otherwise these class attributes don't get recognized for class completion, since we don't set them in the `__init__` explicitly. I should probably address this in a separate PR, will remove them here for now :)